docs/migration/6_0/deleted-api.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "apiElementName": "SomeService",
+    "entryPoint": "@spartacus/feature/core",
+    "migrationComment": "Use class OtherService instead.  (note: this example won't appear in the migration material because the SomeService class does not exist)"
+  },
+  {
+    "apiElementName": "AwesomeClass",
+    "entryPoint": "@spartacus/feature/components",
+    "migrationComment": "This class is not needed anymore.  Things are done in this way instead.  (note: this example won't appear in the migration material because the AwesomeClass class does not exist)"
+  }
+]
```

Comment:
> As these 'generated'-schematics/docs are created, they are not 'linked/mapped' to the actual schematics migration mechanism.

It's not a bad idea.  However, we usually bootstrap the migration schematics one per major release.  It's a short one shot manual operation.  Moreover we still need to bootstrap the schematics manually because there are more schematics and more files involved than these.
The scope of the breaking change tool was rather to focus on the output it is directly responsible for producing and eliminate the annoying copy/past that needed to happen for the generated schematics to be where they should.  
Moreover, the files you mention can potentially be updated by multiple devs if they need to add manually written schematics.
It doesn't seem it's worth it to generate these files automatically, but we can discuss it.